doctoc.js

```diff
@@ -16,17 +16,21 @@ function cleanPath(path) {
   return homeExpanded.replace(/\s/g, '\\ ');
 }
 
-function transformAndSave(files, mode, maxHeaderLevel, title, notitle, entryPrefix, processAll, stdOut) {
+function transformAndSave(files, mode, maxHeaderLevel, title, notitle, entryPrefix, processAll, stdOut, updateOnly) {
   if (processAll) {
     console.log('--all flag is enabled. Including headers before the TOC location.')
   }
+
+  if (updateOnly) {
+    console.log('--update-only flag is enabled. Only updating existing TOC.')
```

Comment:
```suggestion
    console.log('--update-only flag is enabled. Only updating files that already have a TOC.')
```

@AndrewSouthpaw Now sounds concise!